File: MultiLangTest/Sample.java
import java.util.Scanner;

public class Sample {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Prompt user for input
        System.out.print("Enter your age: ");
        int age = scanner.nextInt();

        // Check age validity
        if (age < 0) {
            System.out.println("Invalid age entered.");
        } else {
            System.out.println("Your age is " + age);
        }

        scanner.close();
    }
}

File: MultiLangTest/Sample.js
// Function to add two numbers
function add(a, b) {
    return a + b;
}

// Prompt user for input and parse to integer
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question("Enter first number: ", num1 => {
    readline.question("Enter second number: ", num2 => {
        const result = add(parseInt(num1), parseInt(num2));
        console.log(`Result: ${result}`);
        readline.close();
    });
});
